test/plugins/test_completion.py

```diff
@@ -42,6 +45,8 @@ def test_rope_import_completion(config, workspace):
     assert items is None
 
 
+@pytest.mark.skipif(str2tuple(jedi.__version__) <= str2tuple('14.0'),
```

Comment:
You can compare strings directly, like this

    jedi.__version__ < '0.14.0'

so there's no need of `str2tuple`.

Also, please notice the `<` instead of `<=` here.

Reply:
We cannot compare the strings as that. For example,

<img width="230" alt="Captura de pantalla 2019-07-25 a la(s) 2 23 39 p  m" src="https://user-images.githubusercontent.com/20992645/61902472-d114f200-aee7-11e9-82a6-91869f06e66d.png">

I thought that we shouldn't run those tests with jedi 0.14.0, only with greater versions.

Reply:
We cannot compare version strings as that, in some cases it will not work as expected
<img width="230" alt="Captura de pantalla 2019-07-25 a la(s) 2 23 39 p  m" src="https://user-images.githubusercontent.com/20992645/61902981-0a9a2d00-aee9-11e9-9e61-4df7804d191d.png">

I thought we only wanted to run these tests if jedi version is 0.14.1 or greater

Reply:
You're totally right! Then please use the following function from distutils, which should be more robust than str2tuple:

```python
from distutils.version import LooseVersion

LooseVersion('0.3.0') < LooseVersion('0.14.0')
```